.travis.yml

```diff
@@ -78,44 +107,34 @@ jobs:
               - WOOCOMMERCE_BLOCKS_PHASE=3
         - name: E2E Tests (WP 5.2)
           script:
-              - npm ci
-              - npm run test:e2e:up
-              - composer install
-              - npm run build:e2e-test
               - npm run test:e2e
           env:
               - WP_VERSION=5.2
+              - E2E_TESTS=1
               - WOOCOMMERCE_BLOCKS_PHASE=3
         - name: E2E Tests (WP 5.3)
           script:
-              - npm ci
-              - npm run test:e2e:up
-              - composer install
-              - npm run build:e2e-test
               - npm run test:e2e
           env:
               - WP_VERSION=5.3
+              - E2E_TESTS=1
               - WOOCOMMERCE_BLOCKS_PHASE=3
         - name: E2E Tests (WP 5.4)
           script:
-              - npm ci
-              - npm run test:e2e:up
-              - composer install
-              - npm run build:e2e-test
               - npm run test:e2e
           env:
               - WP_VERSION=5.4
+              - E2E_TESTS=1
               - WOOCOMMERCE_BLOCKS_PHASE=3
         - name: E2E Tests (WP 5.4 with Gutenberg plugin)
           script:
-              - npm ci
-              - npm run test:e2e:up
-              - composer install
-              - npm run build:e2e-test
+              - npm run wp-env run tests-cli wp plugin install gutenberg --activate
```

Comment:
should this be `npm run wp-env run tests-cli "wp plugin install gutenberg --activate"`?

Reply:
yup updated :)